Dockerfile

```diff
@@ -1,5 +1,5 @@
 # BUILDER IMAGE
-FROM python:3.9-slim-bullseye AS builder
+FROM python:3.12-slim AS builder
```

Comment:
Q Is there something in `3.12-slim` that isn't available in `-slim-bullseye`?

Reply:
I updated to explicitly say `3.12-slim-bookworm` to also pin Debian releases.